src/main/java/edu/harvard/iq/dataverse/api/Datasets.java

```diff
@@ -4432,8 +4459,12 @@ public Response getDatasetVersionCitation(@Context ContainerRequestContext crc,
                                               @QueryParam("includeDeaccessioned") boolean includeDeaccessioned,
                                               @Context UriInfo uriInfo,
                                               @Context HttpHeaders headers) {
+        boolean checkFilePerms = false;
+        boolean checkUserPerms = true;
```

Comment:
This variable _checkUserPerms_ is never used.

Reply:
I am removing this, this was a leftover from when I tried to skip checking the user permissions which as the conversation in slack is not what is intended.